calabash.xcodeproj/project.pbxproj

```diff
@@ -7,6 +7,13 @@
 	objects = {
 
 /* Begin PBXBuildFile section */
+		89043ABF1C289A5D0067E22F /* LPConstants.h in Headers */ = {isa = PBXBuildFile; fileRef = 89043ABD1C289A5D0067E22F /* LPConstants.h */; };
```

Comment:
LPContants.h should not be in the Headers.

Can you check your target > Build Phases > Headers section?